use-timescale/extensions/index.md

```diff
@@ -11,52 +11,137 @@ tags: [extensions]
+| [pgcrypto][pgcrypto]                             | Cryptographic functions                                                | -                  |
+| [pgrowlocks][pgrowlocks]                         | Show row-level locking information                                     | -                  |
+| [pgstattuple][pgstattuple]                       | Obtain tuple-level statistics                                          | -                  |
+| [pg_freespacemap][pg_freespacemap]               | Examine the free space map (FSM)                                       | -                  |
+| [pg_prewarm][pg_prewarm]                         | Prewarm relation data                                                  | -                  |
+| [pg_stat_statements][pg_stat_statements]         | Track execution statistics of all SQL statements executed              | For all services   |
+| [pg_trgm][pg_trgm]                               | Text similarity measurement and index searching based on trigrams      | -                  |
+| [pg_visibility][pg_visibility]                   | Examine the visibility map (VM) and page-level visibility info         | -                  |
+| [refint][refint]                                 | Functions for implementing referential integrity (obsolete)            | -                  |
+| [seg][seg]                                       | Data type for representing line segments or floating-point intervals   | -                  |
+| [sslinfo][sslinfo]                               | Information about SSL certificates                                     | -                  |
+| [tablefunc][tablefunc]                           | Functions that manipulate whole tables, including crosstab             | -                  |
+| [tcn][tcn]                                       | Trigger change notifications                                           | -                  |
+| [tsm_system_rows][tsm_system_rows]               | `TABLESAMPLE` method which accepts the number of rows as a limit       | -                  |
+| [tsm_system_time][tsm_system_time]               | `TABLESAMPLE` method which accepts the time in milliseconds as a limit | -                  |
+| [unaccent][unaccent]                             | Text search dictionary that removes accents                            | -                  |
+| [uuid-ossp][uuid-ossp]                           | Generate universally unique identifiers (UUIDs)                        | -                  |
+
+## Third-party extensions
+
+| Extension                                        | Description                                                | Enabled by default    |
+|--------------------------------------------------|------------------------------------------------------------|-----------------------|
+| [bool_plperl][bool_plper]                        | Transform between bool and plperl                          | -                     |
```

Comment:
Those 3 (bool_plperl, jsonb_plperl and old_snapshot) are also postgres built-ins.